packages/rancher-logging/generated-changes/patch/Chart.yaml.patch

```diff
@@ -20,5 +20,4 @@
 +  catalog.cattle.io/ui-component: logging
 +  catalog.cattle.io/provides-gvr: logging.banzaicloud.io.clusterflow/v1beta1
 +  catalog.cattle.io/display-name: "Logging"
-+  catalog.cattle.io/os: linux
```

Comment:
This was the annotation that hid the chart with Windows clusters. It's been removed to reveal the chart now.